### ISSUE TITLE
Build failure with ghc 8.4

### ISSUE THREAD

**Issue by DanBurton** [fpco/inline-c#73]
```
Building library for inline-c-0.6.0.5..
[3 of 9] Compiling Language.C.Types ( src/Language/C/Types.hs, dist/build/Language/C/Types.o )

src/Language/C/Types.hs:106:10: error:
    • No instance for (Semigroup Specifiers)
        arising from the superclasses of an instance declaration
    • In the instance declaration for ‘Monoid Specifiers’
    |
106 | instance Monoid Specifiers where
    |          ^^^^^^^^^^^^^^^^^
```

**Comment by peti** [comment]
Will this issue be fixed?

It's really a shame that `inline-c` does not compile with recent versions of `ghc` because that blocks a whole lot of other packages, too, like `inline-r`. :-(

**Comment by peti** [comment]
Well, there are *two* pull requests readily available, even: https://github.com/fpco/inline-c/pull/74 and https://github.com/fpco/inline-c/pull/66. I don't understand what is going on. :-(

**Comment by bitonic** [comment]
sorry about the delay, i've merged #74 . i didn't get notifications, i believe because i was not a collaborator.

**Comment by bitonic** [comment]
now released as 0.6.0.6
